src/DotLiquid/StandardFilters.cs

```diff
@@ -22,7 +22,7 @@ public static class StandardFilters
     {

```

Comment:
I would expect we should not have any code that now has a comparison to SyntaxCompatibility, but I think there are a few remaining (which I can't comment on directly in the PR for some reason) - e.g. Slice (line 79), ReplaceFirst, Date, etc. These need to be split into a completely separate function (even if that means duplicating code) to make the change clean.

Reply:
I applied same logic to Slice, ReplaceFirst, and RemoveFirst. Leaving Date method alone for now.